data/dockerFiles/Dockerfile.fedora.in

```diff
@@ -0,0 +1,100 @@
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2016 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+
+# XXX: named version (also referenced below for the OBS elements)
+FROM fedora:@DOCKER_FEDORA_VERSION@
+
+# Disable deltarpms.
+#
+# See: https://bugzilla.redhat.com/show_bug.cgi?id=1343114)
+RUN echo deltarpm=0 >> /etc/dnf/dnf.conf
+
+# Ensure packages are current, then install:
+#
+# - a basic development environment
+# - runtime build dependencies and additional general tooling
+#
+# (Note: python3-dnf-plugins-core is required for "dnf config-manager").
+RUN dnf -y update && \
+    dnf -y install \
+        @development-tools \
+        autoconf \
+        automake \
+        bats \
+        check \
+        check-devel \
+        cppcheck \
+        gettext-devel \
+        glib2-devel \
+        json-glib-devel \
+        lcov \
+        libffi-devel \
+        libmnl-devel \
+        libtool \
+        libuuid-devel \
+        pcre-devel \
+        pkgconfig \
+        python3-dnf-plugins-core \
+        sudo \
+        valgrind \
+        zlib-devel \
+		cpan \
+		htop procps-ng \
+		file \
+		perl-TAP-Formatter-HTML.noarch \
+		golang
+
+
+
+# Add the OBS repository
+# XXX: Note the named version of Fedora
+RUN dnf -y config-manager --add-repo \
+    http://download.opensuse.org/repositories/home:clearlinux:preview:clear-containers-2.0/Fedora_24/home:clearlinux:preview:clear-containers-2.0.repo
```

Comment:
There's one more occurence of `24` here that needs to use `@DOCKER_FEDORA_VERSION@`.